k8sdeps/configmapandsecret/configmapfactory_test.go

```diff
@@ -40,6 +40,7 @@ func makeEnvConfigMap(name string) *corev1.ConfigMap {
 			"DB_USERNAME": "admin",
 			"DB_PASSWORD": "somepw",
 		},
+		BinaryData: map[string][]byte{},
```

Comment:
Can you think of a way not adding this line when `BinaryData` is empty?

Reply:
the BinaryData map is now nil until/unless the generator finds a binary file